LICENSE

```diff
@@ -181,6 +181,33 @@ Neovim's license follows:
       incurred by, or claims asserted against, such Contributor by reason
       of your accepting any such warranty or additional liability.
 
+	 END OF TERMS AND CONDITIONS
+
+	 APPENDIX: How to apply the Apache License to your work.
+
+	    To apply the Apache License to your work, attach the following
+	    boilerplate notice, with the fields enclosed by brackets "[]"
+	    replaced with your own identifying information. (Don't include
+	    the brackets!)  The text should be enclosed in the appropriate
+	    comment syntax for the file format. We also recommend that a
+	    file or class name and description of purpose be included on the
+	    same "printed page" as the copyright notice for easier
+	    identification within third-party archives.
+
+	 Copyright [yyyy] [name of copyright owner]
```

Comment:
See https://github.com/apple/swift/blob/master/LICENSE.txt

Swift includes it still because its part of the apache license.